learn/cases.html

```diff
@@ -81,6 +81,8 @@ <h1>OBL Cases</h1>
           </a>
         </td>
         <td class="caseImageColumn">
+          {% include cubicle_embed.html title=title alg=case.default_alg.alg %}
```

Comment:
After discussion with Calvin and Ben online, we decided that this _probably_ isn't necessary for v1 if this is prohibitively slow, but especially with the white/yellow swap in this pr, the old static images will look increasingly wrong.

Especially in prep for custom color schemes (#22), dynamic image loading will likely be better; as long as this page loads "fast enough" (I think mostly "first image populates quickly" combined with "changing table filter doesn't cause image refreshes", we'll be fine), this direction will probably be better, and we should work on faster image loading in #27.

Reply:
Too slow, reverted :(